woocommerce-payments.php

```diff
@@ -8,8 +8,8 @@
  * Woo: 5278104:bf3cf30871604e15eec560c962593c1f
  * Text Domain: woocommerce-payments
  * Domain Path: /languages
- * WC requires at least: 7.1
- * WC tested up to: 7.3.0
+ * WC requires at least: 7.3
+ * WC tested up to: 7.5.0
```

Comment:
The latest WC release right now is [7.5.1](https://github.com/woocommerce/woocommerce/releases/tag/7.5.1).

Reply:
Updated in d7d98c3. Thanks for checking.